examples/nidaqmx/analog-input.py

```diff
@@ -51,46 +51,41 @@
 task = None
 
 
-def raise_if_error(response):
-    """Raise an exception if an error was returned."""
-    if response.status != 0:
-        response = client.GetErrorString(
-            nidaqmx_types.GetErrorStringRequest(error_code=response.status)
-        )
-        raise Exception(f"Error: {response.error_string}")
-
-
 try:
+    def check_for_warning(response):
+        """Print to console if the status indicates a warning."""
+        if response.status > 0:
+            warning_message = client.GetErrorStringResponse(
```

Comment:
You've likely done this in all the files. I won't mention it elsewhere, just verify it globally in your PR.

Reply:
Switched to `GetErrorString` throughout the PR.